<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTaskDialog</class>
 <widget class="QDialog" name="AddTaskDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Task</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name of the project</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Task</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Task"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>FIO of implementer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="FIOImplem">
       <property name="placeholderText">
        <string>First Middle Last</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>FIO of manager</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="FIOManeger"/>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="DateAssig"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Deadline</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="DateDeadline"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Task complited(date)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDateEdit" name="DateTaskCompl"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Assignment date</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTaskDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTaskDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
